src/components/editor/Shards/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { useEditorStore_persistedDraftId } from 'components/editor/Store/hooks';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import WrapperWithHeader from 'components/shared/Entity/WrapperWithHeader';
+import ShardsDisable from './Disable';
+
+interface Props {
+    renderOpen: boolean;
```

Comment:
It does not control if it is open - only if it is rendered open.

Reply:
It looks like my edit to the original comment didn't save, darn. There was supposed to be a list of all the variations of open-related component properties. `renderOpen` is used as a prop elsewhere in the codebase, but I feel it may be beneficial to keep an eye on the names of like properties (e.g., `forceOpen`, `renderOpen`, `defaultOpen`, etc.) at this stage.

Reply:
Gotcha. So those first two are totally different of `forceOpen` and `renderOpen`... but yeah I get the sentiment. Might end up always using the controlled version of MUI components so we can just control stuff manually.